docker/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+# Docker Compose configuration for StarryOS development
+# Note: This is an optional convenience tool for development.
+# You can also use Docker commands directly (see README.md)
+
+services:
+  starryos:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    image: starryos:latest
```

Comment:
Image name suggestion: `localhost/starry-env`

Or docker compose will be confused to try to pull the image from Docker Hub first. See:
```
 ! starry Warning pull access denied for starryos, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

Using a `localhost` prefix let it fail to pull the image and start building immediately:
```
 ! starry Warning Get "http://localhost/v2/": dial tcp [::1]:80: connect: connection refused
```

And why `starry-env`: `starryos` may confuse users to think this image is directly a StarryOS deployment.